app/src/main/java/org/simple/clinic/newentry/PatientEntryEffectHandler.kt

```diff
@@ -0,0 +1,15 @@
+package org.simple.clinic.newentry
+
+import com.spotify.mobius.rx2.RxMobius
+import io.reactivex.ObservableTransformer
+import org.simple.clinic.util.scheduler.SchedulersProvider
+
+object PatientEntryEffectHandler {
+  fun createEffectHandler(
```

Comment:
Since we will be accessing this method through the `PatientEntryEffectHandler` object, I feel the `EffectHandler` suffix to the method name is redundant. What do you think about calling it just `create()` instead?

Reply:
Noted, will make this change as part of the next PR because it is 10+ commits ahead.